package.json

```diff
@@ -164,6 +166,7 @@
     "build-chromeapp": "webpack --config ./scripts/chromeapp.webpack.js",
     "build-js": "webpack --config ./scripts/browser.webpack.js",
     "build-js-worker": "webpack --config ./scripts/worker.webpack.js",
+    "build-d-ts": "rm -rf dist/**/*.d.ts  &&  tsc --emitDeclarationOnly",
```

Comment:
I recommend using something like `npx rimraf -g dist/**/*.d.ts` to ensure all scripts are cross-platform.